sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/global-namespace/build.sbt

```diff
@@ -22,8 +22,6 @@ npmDevDependencies in Compile += "expose-loader" -> "0.7.1"
 // as expected by the scalajs-react facade
 webpackConfigFile := Some(baseDirectory.value / "webpack.config.js")
 
-// Needed to make scalajs-bundler actually use the custom config file in fastOptJS
-enableReloadWorkflow := false
```

Comment:
Because the reload workflow is always disabled for `fullOptJS`. We should add a note about that.